cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -145,9 +145,34 @@
         border-bottom: 1px solid @horizontal-rule;
         margin-bottom: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
     }
+
+    &__email {
```

Comment:
It doesn't have to happen here, but this probably should be a organism.

Reply:
Should just the email inset be an organism?